Assets/Scripts/Managers/LevelController.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LevelController : MonoBehaviour
+{
+    public static LevelController lController;
+
+    //public transform playerSpawn;
+
+    void Awake()
+    {
+        if (lController != null)
+        {
+            Destroy(lController.gameObject);
+            lController = this;
+        }
+        else
+        {
+            lController = this;
```

Comment:
You do this in the if and else, so just pull it outside of it an only have an if for the destroy